### ISSUE TITLE
http://www.gutenberg.lib.md.us mirror working

### ISSUE THREAD

**Issue by jrdnbradford** [ropensci/gutenbergr#82]
`gutenberg_get_mirror` has the following:

```R 
 # ...
  if (parsed$domain == "www.gutenberg.lib.md.us") {
    # Broken mirror. PG has been contacted. For now, replace:
    parsed$domain <- "aleph.gutenberg.org" # nocov
  }
# ...
```

The http://www.gutenberg.lib.md.us mirror seems to be up and running again. However, I also noticed this mirror is not listed on the [official mirrors page](https://www.gutenberg.org/MIRRORS.ALL). 

There are a few things I think could be worth doing based on this:

1. Remove the above code so that this mirror can be used.
2. Document as part of the `gutenberg_get_all_mirrors` function that it retrieves _officially listed_ mirrors and that _other_ mirrors may exist.
3. Perhaps add an unexported function to test whether or not a particular mirror is indeed a mirror and is working. Maybe by attempting to request `GUTINDEX.zip` from the mirror, and providing some sort of feedback? Maybe this should be a different PR than 1 and 2 if we want to do this.

@jonthegeek let me know you think here.

**Comment by jonthegeek** [comment]
Yeah, go ahead and do 1 and 2, and write up 3 as a separate issue.
